plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/connection/DBPDriver.java

```diff
@@ -105,6 +105,14 @@ public interface DBPDriver extends DBPNamedObject
     @Nullable
     String getPropertiesWebURL();
 
+    /**
+     * getDriverDocumentationURL
+     *
+     * @return DriverDocumentationURL
+     */
```

Comment:
Please don't write such useless javadocs. It's better to just omit it completely.